.travis.yml

```
@@ -14,7 +14,7 @@ before_install:
 # download large files
 - pip install awscli
 install:
-- pip install -e ".[test]"
+- travis_wait pip install -e ".[test]"
```

Might consider including `--no-cache-dir`, e.g.
```suggestion
- travis_wait pip install --no-cache-dir -e ".[test]"
```
 to see if it actually reduces torch download time instead of just increasing Travis runtime